aliases/available/git.aliases.bash

```diff
@@ -18,6 +18,7 @@ alias gbd='git branch -d'
 alias gbm='git branch -m'
 alias gbt='git branch --track'
 alias gdel='git branch -D'
+alias gbup='git log --branches --not --remotes --no-walk --decorate --oneline' # FROM https://stackoverflow.com/questions/39220870/in-git-list-names-of-branches-with-unpushed-commits
```

Comment:
I believe we keep these aliases alphabetized within each sub-group.
So you can move this up between `gbt` and and `gdel`.
Other than that, looks great !

Reply:
Sure will do that in the UI now!